src/Mailgun/Api/Suppressions/Bounce.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Api;
+
+use Mailgun\Assert;
+use Mailgun\Resource\Api\Suppressions\Bounce\CreateResponse;
+use Mailgun\Resource\Api\Suppressions\Bounce\DeleteResponse;
+use Mailgun\Resource\Api\Suppressions\Bounce\IndexResponse;
+use Mailgun\Resource\Api\Suppressions\Bounce\ShowResponse;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class Bounce extends HttpApi
+{
+    use Pagination;
+
+    /**
+     * @param string $domain
+     *
+     * @return IndexResponse
+     */
+    public function index($domain, $limit = 100)
+    {
+        Assert::notEmpty($domain);
+        Assert::range($limit, 1, 10000, 'Limit parameter must be between 1 and 10000');
+
+        $response = $this->httpGet(sprintf('/v3/%s/bounces', $domain));
+
+        return $this->safeDeserialize($response, IndexResponse::class);
+    }
+
+    /**
+     * @param string $domain
+     * @param string $address
+     *
+     * @return ShowResponse
+     */
+    public function show($domain, $address)
+    {
+        Assert::notEmpty($domain);
+        Assert::notEmpty($address);
+        $response = $this->httpGet(sprintf('/v3/%s/bounces/%s', $domain, $address));
+
+        return $this->safeDeserialize($response, ShowResponse::class);
+    }
+
+    /**
+     * @param string    $domain
+     * @param string    $address
+     * @param string    $code      optional
+     * @param string    $error     optional
+     * @param \DateTime $createdAt optional
```

Comment:
This is not a blocker, but if we're making these parameters optional (due we're setting them as `null` by default) then ideally the annotations should provide/allow this option too, so:
```
* @param string|null    $code      optional
* @param string|null    $error     optional
* @param \DateTime|null $createdAt optional
```
Don't forget to add the extra blank spaces on the other 2 `@param` above these ones! :-)